README.md

```diff
@@ -8,6 +8,28 @@ This is project is structured so that the developer can choose to set it up with
 <br />
 
 # **PROJECT SETUP**
+## **Cloning Project**
+Clone this repo and change directory into the project root folder by running the following commands:
+
+```bash
+git clone https://github.com/KanaryStack/aliascheck.git
+```
+>NOTE:
+>If you have setup an SSH connection to this repo, you can run this command instead:
+>```bash
+>git clone git@github.com:KanaryStack/aliascheck.git
+>```
+
+and go to the project root folder like so:
+```bash
+cd aliascheck
+```
+then switch to last stable branch with Flask backend:
+```bash
+git checkout 940bc4d36f8a34e65373e080024cf98d97b4cf52
```

Comment:
I will recommend we only merge(probably will need further update) after a Node PR is merged to master. 
```suggestion
git checkout main
```
This will allow us to give specific contribution/setup instruction to the contributor/user.

Reply:
Once the PR is merged, we can update this README apprpriately https://github.com/KanaryStack/aliascheck/pull/26